apps/web/src/components/AssessmentStepTracker/utils.ts

```diff
@@ -32,7 +33,7 @@ import {
 } from "~/utils/poolCandidate";
 
 export type CandidateAssessmentResult = {
-  poolCandidate: PoolCandidate;
+  poolCandidate: AssessmentStepTracker_CandidateFragment;
   decision: NullableDecision;
 };
```

Comment:
Pulls the type from the graphql file so avoids being a circular reference. Enables better type safety with minimal refactor.